### DISCUSSION TITLE
formily/next中FormLayout等使用了ResizeObeserver特性的组件在低版本浏览器中兼容性

### DISCUSSION THREAD

**Discussion by HuColin** [alibaba/formily#2959] [Q&A 2.x] [answered]
在formily/next的FormLayout组件，FormGrid组件中使用了ResizeObserver特性
![image](https://user-images.githubusercontent.com/16102181/158984144-5e01a69b-d7d7-42df-a0bc-72a932e688dc.png)
但是低版本浏览器不存在ResizeObserver这个对象，这里似乎没有做兼容处理，会使得页面直接报错
![image](https://user-images.githubusercontent.com/16102181/158984389-63cd50f6-cdc7-4af3-863d-d9c6893cbb88.png)
[CanIUse-ResizeObeserver](https://caniuse.com/?search=ResizeObserver)

**Comment by janryWang** [comment, marked as answer]
本身formily就不支持ie，你要兼容ie就换方案吧

**Reply by HuColin** [reply]
老版本的Safari不算是IE吧😂

**Reply by janryWang** [reply]
老版本safari支持Proxy吗？还有，建议自己项目引入polyfill，而不是让库来引入，库更多是for标准，如果默认引入polyfill，就会导致未来代码体积一直都是很大
